app/services/protocol_importers/build_protocol_from_client_service.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module ProtocolImporters
+  class BuildProtocolFromClientService
+    extend Service
+
+    attr_reader :errors, :pio_protocol
+
+    def initialize(protocol_client_id:, protocol_source:, user_id:, team_id:)
+      @id = protocol_client_id
+      @protocol_source = protocol_source
+      @user = User.find user_id
+      @team = Team.find team_id
+      @errors = {}
+    end
+
+    def call
+      return self unless valid?
+
+      # TODO: check for errors
+      api_response = api_client.single_protocol(id: @id)
+      normalized_hash = normalizer.load_protocol(api_response.parsed_response)
+
+      pio = ProtocolImporters::ProtocolIntermediateObject.new(normalized_json: normalized_hash,
+                                                              user: @user,
+                                                              team: @team)
+
```

Comment:
I think you are missing a line here:
`@pio_protocol = pio.build`

Reply:
Nice catch!